sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeExtractors.scala

```diff
@@ -64,7 +64,7 @@ object ExtractValue {
 
       case (_: ArrayType, _) => GetArrayItem(child, extraction)
 
-      case (MapType(kt, _, _), _) => GetMapValue(child, extraction)
+      case (MapType(_, _, _), _) => GetMapValue(child, extraction)
```

Comment:
Can I ask what is the intention for this change?
Seems like maybe it is not required to update an error class name??

Reply:
Are you asking about this particular line of code or the whole change? 
If about the change itself - then at least it improves code readability IMHO

Reply:
Oh, yeah I got it it's just code clean-up.
Sounds good